scanner/updater/export.go

```diff
@@ -0,0 +1,182 @@
+package updater
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/klauspost/compress/zstd"
+	"github.com/quay/claircore/alpine"
+	"github.com/quay/claircore/aws"
+	"github.com/quay/claircore/debian"
+	"github.com/quay/claircore/libvuln/driver"
+	"github.com/quay/claircore/libvuln/jsonblob"
+	"github.com/quay/claircore/libvuln/updates"
+	"github.com/quay/claircore/oracle"
+	"github.com/quay/claircore/photon"
+	"github.com/quay/claircore/rhel"
+	"github.com/quay/claircore/suse"
+	"github.com/quay/claircore/ubuntu"
+	"github.com/quay/claircore/updater/osv"
+	"github.com/quay/zlog"
+	"github.com/stackrox/stackrox/scanner/v4/updater/manual"
+	"golang.org/x/time/rate"
+)
+
+// Export is responsible for triggering the updaters to download Common Vulnerabilities and Exposures (CVEs) data
+// and then outputting the result as a zstd-compressed file with .ztd extension
+func Export(ctx context.Context, outputDir string) error {
```

Comment:
I think the cvss enricher data is missing

Reply:
I have reviewed the contents of https://github.com/quay/claircore/blob/main/enricher/cvss/cvss.go, and it appears to have a different approach compared to how the updater works. I don't have a picture how to move forward with this data but I created a ticket for it: https://issues.redhat.com/browse/ROX-18455